erdos/src/dataflow/graph/internal_graph.rs

```diff
@@ -146,11 +146,11 @@ impl Clone for Box<dyn DriverStreamSetupHook> {
     }
 }
 
->>>>>>> 7c793179566ed40571c04d378b15aef0c690b41f
 /// The abstract graph representation of an ERDOS program defined in the driver.
 ///
-/// The abstract graph is compiled into a [`JobGraph`], which ERDOS schedules and executes.
-#[derive(Default)]
+/// The abstract graph is compiled into a JobGraph, which ERDOS schedules and executes.
+/// TODO: Make this struct private.
+#[derive(Default, Clone)]
```

Comment:
The `InternalGraph` shouldn't derive `Default` since the `name` argument is required by the constructor.